src/openttd.cpp

```diff
@@ -330,6 +330,24 @@ static void ShutdownGame()
 	UninitFreeType();
 }
 
+/**
+* Get the introduction game landscape specific save file.
+*/
+static std::string GetIntroGameSaveFile(LandscapeID landscape)
+{
+	switch (landscape)
+	{
+	case LT_ARCTIC:
```

Comment:
As per the convention, please intend case blocks, and keep { in the same line
`switch (landscape) {`

Reply:
Has there been thought about adding a format file (.clang-format for example) to the repo?

Reply:
I've personally tried it a few times. OTTD's codestyle is just too "unique" to be applied sanely, there's too much manually formatted code (the window definitions are the obvious examples)